content/blog/2020/2020-10-01-document-embedders.md

```diff
@@ -0,0 +1,73 @@
++++
+author = "Primož Godec and Nikola Djukič"
+date = "2020-10-01"
+draft = false
+title = "How to identify fake news by document embeddings"
+type = "blog"
+thumbImage = "/blog_img/2020/2020-10-01-document-embedding-widget.png"
+frontPageImage = "/blog_img/2020/2020-10-01-document-embedding-widget.png"
+categories = ["text mining", "corpus", "classification"]
+shortExcerpt = "New Document embedder widget and its use for classification"
+longExcerpt = "Presenting document embeddings widget and how to identify fake news."
+x2images = false  # true if using retina screenshots, else false
++++
+
+Text is by default described by the sequence of character. Since every machine learning algorithm needs numbers, we need to transform it into vectors of real numbers before we can continue with the analysis. To do this, we can use various approaches. Orange currently offers Bage of words approach and now also Document embedding by FastText. 
```

Comment:
If I'm not mistaken, fastText should be written with lowercase f.

Reply:
You are right.